pkg/tsdb/testdata/v3io.yaml.template

```diff
@@ -15,7 +15,5 @@ verbose: "warn"
 # Default timeout 10 seconds
 timeout: 10
 
-workers: 8
```

Comment:
Why delete it from the template, though?

Reply:
no workers means default, i.e workers=1

Reply:
I would leave it commented out so users will see the option to control it